package.json

```diff
@@ -38,5 +38,6 @@
     "react-dom": "^15.0.2",
     "react-hot-loader": "^3.0.0-alpha.8",
     "showdown": "^1.3.0"
-  }
+  },
+  "main": "src/index.js"
```

Comment:
Once this is up on npm this will enable proper importing by package name.

`require('reactcards')`